src/components/fields/image-upload/types.ts

```diff
@@ -90,6 +90,12 @@ export interface IUpdateImageStatus {
 	errorMessage?: string | undefined;
 }
 
+export interface IUploadedImage {
+	dataURL: string;
+	fileName: string;
+	uploadResponse: string;
```

Comment:
`uploadResponse` can be anything returned by the upload api

```suggestion
	uploadResponse?: any;
```

Reply:
updated